src/main/java/seedu/kitchenhelper/command/CookRecipeCommand.java

```diff
@@ -34,16 +35,16 @@ public String cookRecipe(ArrayList<Ingredient> ingredientList, ArrayList<Recipe>
             throws KitchenHelperException {
         // checks if the specified recipe given by user exists
         int indexOfRecipe = checkIfRecipeExist(recipeList);
-        if (indexOfRecipe != recipeList.size()) {
-            throw new KitchenHelperException("The given recipe name does not exist");
+        if (indexOfRecipe > recipeList.size()) {
+            return "The given recipe name does not exist!";
         }
         System.out.println("Kitchen Helper is trying to cook!");
         Recipe recipeToBeCooked = recipeList.get(indexOfRecipe - 1);
 
         if (checkForSufficientIngredient(ingredientList, recipeToBeCooked)) {
             deductIngredients(ingredientList, recipeToBeCooked);
         } else {
-            throw new KitchenHelperException("There are insufficient/ missing ingredients!");
+            return "There are insufficient/ missing ingredients!";
```

Comment:
Perhaps you can use non-magic string literals?

Reply:
Thank you for the feedback, I have edited some of them into non-magic string literals!